.gitignore

```diff
@@ -1,3 +1,6 @@
+# Go embed files?
+*.go-e
```

Comment:
not a bad thing I assume, but what is this??

Reply:
I actually have no idea, but I had tons of them in my tree after doing the bazel build.